src/content/docs/browser/new-relic-browser/configuration/experimental-features.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: 'Experimental Features'
+metaDescription: "Opt-in to use experimental features in New Relic Browser before they are generally available."
+redirects:
+  - /docs/browser/new-relic-browser/configuration
+freshnessValidatedDate: never
+---
+
+New Relic Browser features are exposed to customers in a controlled manner to ensure they are stable and reliable. However, some features are exposed before they generally available. These features are called **experimental features**. To access these features, you must opt in to use them.
```

Comment:
New Relic Browser features are exposed to customers in a controlled manner to ensure stability and reliability. However, some features are made available before they reach general availability. These are known as experimental features. To access them, you must opt in.